Cesium.Compiler/stdlib/conio.h

```diff
@@ -0,0 +1,8 @@
+__cli_import("Cesium.Runtime.ConioFunctions::KbHit")
+int kbhit(void);
+
+__cli_import("Cesium.Runtime.ConioFunctions::GetCh")
+int getch(void);
+
+__cli_import("Cesium.Runtime.ConioFunctions::Cls")
+void cls(void);
```

Comment:
I wasn't able to find this function in any `conio.h` implementation I searched. Where is this from?

Reply:
That’s my replacement for `system(“cls”)`. Looking backwards I better provide system function with this functionality. Or maybe you know better way to clear console screen?